js/script.js

```diff
@@ -0,0 +1,44 @@
+//
+// Menu mobile toggle active
+//
+
+const h = document.querySelector('header');
+const m = document.querySelector('.menu-bars');
+const i = document.querySelector('.menu-bars i');
```

Comment:
kindly it is highly recommended to use descriptive variable names, that will help a lot for you or another colleague to understand your code easier

Reply:
Thank you for teaching me that. I will try not to repeat this error.